preprocess/analysis/mapping.py

```diff
@@ -0,0 +1,49 @@
+"""
+@Author: Scarlet Nguyen 
```

Comment:
Why is the author Scarlet when you wrote this

Reply:
I've removed Scarlet's code from this PR, she is going to handle pushing this code. Going to mark these comments as resolved as Scarlet has seen them.